rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -97,4 +125,54 @@ impl Run {
     pub fn attempt_count() -> u32 {
         LIVESPLIT_STATE.lock().unwrap().timer.run().clone().attempt_count()
     }
+    pub fn save_splits(path: impl AsRef<Path>) -> Result<(), SplitsSaveError> {
+        let state = LIVESPLIT_STATE.lock().unwrap();
+        let path = path.as_ref();
+        let filename = path
+            .file_name().expect("Could not get filename")
+            .to_str().expect("Could not convert filename to &str")
+            .to_owned();
+        let filename = sanitize_filename::sanitize(filename);
+        let valid_splits = VALID_SPLITS.lock().unwrap();
+        let path = match path.is_relative() || valid_splits.contains(&path.to_path_buf()) {
```

Comment:
No need to convert to an owned `PathBuf`: <https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=bce420dcce747580b8f575e8b0e2fc8c>